package.json

```diff
@@ -19,12 +19,12 @@
     "htmlparser2": "~3.7.1",
     "istanbul": "^0.3.2",
     "marked": "~0.3.2",
-    "react": "~0.12.0",
+    "react": "~0.13.0",
     "rimraf": "~2.2.8",
     "testit": "~1.2.0"
   },
   "peerDependencies": {
-    "react": "~0.12.0"
+    "react": ">=0.12.0 <0.14.0"
```

Comment:
Just `~0.13` will no include 0.12. `>=0.12 ~0.13` is more confused, because it mix different types (is it even will works?)